tests/test_server_basic.py

```diff
@@ -1393,3 +1394,60 @@ def test_get_query_products_exception(dispatcher_live_fixture):
     print("jdata : ", jdata)
 
     assert jdata['exit_status']['message'] == 'InternalError()\nfailing query\n'
+
+
+@pytest.mark.test_drupal
+def test_product_gallery_post_article(dispatcher_live_fixture):
+    server = dispatcher_live_fixture
+
+    logger.info("constructed server: %s", server)
+
+    # send simple request
+    # let's generate a valid token
+    token_payload = {
+        **default_token_payload,
+        "roles": "general, unige-hpc-full",
```

Comment:
we should add a role to regulate gallery posting. something like `post-product-gallery`

Reply:
As agreed in [https://github.com/oda-hub/doc-multi-user/pull/9](https://github.com/oda-hub/doc-multi-user/pull/9), this will be `gallery poster`

Reply:
I am not too happy with `poster` since it's also a noun but why not, not a big deal.

Reply:
we still have time to change to something else [https://github.com/oda-hub/doc-multi-user/pull/9](https://github.com/oda-hub/doc-multi-user/pull/9) is not closed yet